Assets/MirageXR/Tests/NewUI/StepsListView_v2.cs

```diff
@@ -54,8 +54,8 @@ public class StepsListView_v2 : BaseView
     public override void Initialization(BaseView parentView)
     {
         base.Initialization(parentView);
-        _inputFieldActivityName.onValueChanged.AddListener(OnActivityNameChanged);
-        _inputFieldActivityDescription.onValueChanged.AddListener(OnActivityDescriptionChanged);
+        _inputFieldActivityName.onEndEdit.AddListener(OnActivityNameChanged);
+        _inputFieldActivityDescription.onEndEdit.AddListener(OnActivityDescriptionChanged);
```

Comment:
if you changed event, pleace change `OnActivityNameChanged` the name to `OnActivityNameEndEdit` too.
same with OnActivityDescriptionChanged.